docs/guides/dns/cloudflared.md

```diff
@@ -84,6 +84,13 @@ Edit configuration file by copying the following in to `/etc/default/cloudflared
 CLOUDFLARED_OPTS=--port 5053 --upstream https://1.1.1.1/dns-query --upstream https://1.0.0.1/dns-query
 ```
 
+If you're running cloudflared on different host than pi-hole, you can add listening address to all IPs:
+
+```bash
+# Commandline args for cloudflared, using Cloudflare DNS
+CLOUDFLARED_OPTS=--port 5053 --upstream https://1.1.1.1/dns-query --upstream https://1.0.0.1/dns-query --address 0.0.0.0
```

Comment:
Wouldn't it be better to add only Pihole's address here?

Reply:
for security yes, but for convenience I used 0.0.0.0. I will add some warning.

Reply:
more info added.